library/packages/src/modules/PackageSystem.rb

```diff
@@ -347,12 +348,15 @@ def Installed(package)
       # Unfortunately, initializing Pkg reads the RPM database...
       # so we must avoid it.
       # added --whatprovides due to bug #76181
-      rpm_command = "/usr/bin/rpm -q --whatprovides #{package.shellescape}"
-      output = SCR.Execute(path(".target.bash_output"), rpm_command)
-      log.info "Query installed package with '#{rpm_command}' and result #{output.inspect}"
+      # Use Yast::Execute to prevent false positives (boo#1137992)
+      rpm_command = ["/usr/bin/rpm", "-q", "--whatprovides", package.shellescape]
```

Comment:
here shellescape is wrong as cheetah does not use shell for command. So wrong package name will be passed.